### PR TITLE
feat(yield): use lp-token in widget

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

1. Added pool info displaying (click on info icon)
2. Bound lp-tokens to yield widget (now you can select it and potentially trade)
3. Generalized lp-token logo displaying
4. Bound search input to lp-token list

image

image

# To Test

1. You should be able to select lp-token in Yield widget
2. If there is more than one lp-token with the same symbol, then token address should be used in URL
3. Pool page should be displayed (design is not finished yet! UI input from @fairlighteth)
4. APR should be displayed in token selector only for AAVE/WETH token (hardcoded)

### CODE REVIEW

**Comment by elena-zh** [comment]
I've been able to open the preview link in the #5014 PR and do some tests. However, I believe, not all the changes described here are available for me to test there. 
For now: 
1. It would be great to hide balances loader to a not connected state
balance load not connected
2. (might be not related to this PR) I'm on the Yield tab, selected some tokens to trade. When I press on the 'Yield' once again, the tokens selection is cleared out
3. Impossible to pick a token from a list, no details page --> I believe this is due to build failure in the current PR.

**Comment by shoom3301** [comment]
@elena-zh sorry, fixed the build

**Comment by elena-zh** [comment]
Hey @shoom3301 , in addition to issues I reported in https://github.com/cowprotocol/cowswap/pull/5013#issuecomment-2427295519: 

1. When there is nothing to display for a pool details, I think it would be nice to hide USD symbol as well and show dash instead
no data - dashes
2. (not sure if we can do anything here) When a pool token contains M token, it is not displayed in the token name (just` /WETH`)
maker
Can we improve it from our side somehow? 
3. (not sure about it) Maybe we should always show pool addresses in the URL instead of token symbols? 
address insead
This is how Balancer does 
image

Thanks

**Comment by shoom3301** [comment]
@elena-zh 
>It would be great to hide balances loader to a not connected state

Fixed

>might be not related to this PR) I'm on the Yield tab, selected some tokens to trade. When I press on the 'Yield' once again, the tokens selection is cleared out

Fixed

>When there is nothing to display for a pool details, I think it would be nice to hide USD symbol as well and show dash instead

Fixed

>(not sure if we can do anything here) When a pool token contains M token, it is not displayed in the token name

Unfortunately it is what we receive from portals.fi. We could improve that, but I would leave this fix for the last

>Maybe we should always show pool addresses in the URL instead of token symbols?

Good idea, added it

**Comment by shoom3301** [comment]
@elena-zh I also fixed some issues from [this PR](https://github.com/cowprotocol/cowswap/pull/5005):

>The lists are not displayed for geo-blocked area. Asked about it [here](https://cowservices.slack.com/archives/C07M5DZHMHC/p1729252605388729). But as for the current behavior it makes this feature not accessible from there.

Fixed

>It is impossible to select a token from a list: nothing happens when I click on it

Fixed

>Search for a token is not working on newly added tabs

Fixed

**Comment by elena-zh** [comment]
Hey @shoom3301 , great!

Still I don't see mocked data, but it is not a big deal as it can be tested later. 
As for current changes: 
1. Not connected: I think if we don't have anything to display, we should show dash like for APR or show nothing in both columns. WDYT? 
dash or 0
2. I can see tokens balances being **not connected** on:
 - account pages
 - in every form when I pick a token
balance tokens page
balances not con
not con

**Comment by shoom3301** [comment]
@elena-zh thanks!
1. Fixed
2. I couldn't reproduce. Can you record a video please? Or let's make a call

**Comment by shoom3301** [comment]
@elena-zh thanks!
Fixed 2nd as well

**Review by elena-zh — APPROVED** [review]
Thank you!

**Review by alfetopito — APPROVED** [review]

File: libs/tokens/src/pure/TokenLogo/index.tsx [hunk 1]
```diff
@@ -83,25 +121,46 @@ export function TokenLogo({ logoURI, token, className, size = 36, sizeMobile }:
...
+          <div>
+            <TokenLogo noWrap token={tokensByAddress[token.tokens?.[0]]} size={size} sizeMobile={sizeMobile} />
+          </div>
+          <div>
+            <TokenLogo noWrap token={tokensByAddress[token.tokens?.[1]]} size={size} sizeMobile={sizeMobile} />
+          </div>
+        </LpTokenWrapper>
+      </TokenLogoWrapper>
+    )
+  }
```
Comment (anxolin) [inline libs/tokens/src/pure/TokenLogo/index.tsx:149]: This is super cool!!!! so we use the same component that you tough it to represent pools!

**Pull request merged**